src/systems/ec/specs/components/social-media-share/demo/data.js

```diff
@@ -0,0 +1,96 @@
+// Simple content for demo
+module.exports = {
+  description: 'Share this page',
+  links: [
+    {
+      href: '/example',
+      label: 'Twitter',
+      className: 'ecl-social-media-share__link--twitter',
+      variant: 'standalone',
+      iconPosition: 'before',
+      icon: [
+        {
+          shape: 'twitter',
+          size: 'xl',
+        },
+        {
+          shape: 'twitter_hover',
+          size: 'xl',
+        },
+      ],
+    },
+    {
+      href: '/example',
+      label: 'Facebook',
+      className: 'ecl-social-media-share__link--facebook',
+      variant: 'standalone',
+      iconPosition: 'before',
+      icon: [
+        {
```

Comment:
the problem with this structure is that you have to remember that the first item is the default icon and the second icon is the icon on hover... but what happens if you add a third icon?